Gemfile

```diff
@@ -0,0 +1,33 @@
+source "https://rubygems.org"
```

Comment:
This is just the default Gemfile for a jekyll site which is what github pages supports. You can run this site locally for yourself and see that it immediately redirects to the jobs site.